src/pages/Checklist/index.tsx

```diff
@@ -518,47 +623,65 @@ export const Checklist = () => {
           <CheckBox
             label={
               <Text className="checkbox-label">
-                <FormattedMessage
-                  defaultMessage="I've forwarded the necessary ports to the correct machine(s)
-                    from my router (only open the ports that apply to your installation)."
-                />
+                <FormattedMessage defaultMessage="I've forwarded the necessary ports to the correct machine(s) from my router for both my EL and CL client (only open the ports that apply to your installations)." />
               </Text>
             }
           />
```

Comment:
Might be worth linking out to some more info on firewalls and port forwarding here